ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -643,6 +649,30 @@ tests:
     env:
       ALLOW_CONVERGED_FLOW: "True"
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-multiarch-ocp-411
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate-4-11-multi
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+    env:
+      ASSISTED_ZTP_CONFIG: |
+        SPOKE_CONTROLPLANE_AGENTS=3
+      NUM_EXTRA_WORKERS: "3"
+    workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-multiarch-ocp-411-sno
```

Comment:
from above jobs it seems we usually put number of nodes (sno/3masters) before the ocp version
see ``e2e-ai-operator-ztp-ipv4v6-3masters-ocp-410``

Reply:
Fixed